bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/components/UIComponentSitemapProvider.java

```diff
@@ -321,11 +322,14 @@ private void addWidgetMappings(EList<Mapping> mappings, UIComponent component) {
             if (component.getConfig().get("mappings") instanceof Collection<?>) {
                 for (Object sourceMapping : (Collection<?>) component.getConfig().get("mappings")) {
                     if (sourceMapping instanceof String) {
-                        String cmd = sourceMapping.toString().split("=")[0].trim();
-                        String label = sourceMapping.toString().split("=")[1].trim();
+                        String[] splittedMapping = sourceMapping.toString().split("=");
```

Comment:
The proper english would be `splitMapping` here.

Reply:
Changed